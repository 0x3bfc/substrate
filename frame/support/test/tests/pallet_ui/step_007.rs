#[frame_support::pallet]
mod pallet {
	#[pallet::config]
	pub trait Config {}
}

fn main() {
}
