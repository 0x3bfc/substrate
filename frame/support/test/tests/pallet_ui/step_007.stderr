error: Invalid pallet::trait, expect explicit `frame_system::Config` as supertrait, found none. To disable this check, use `#[pallet::disable_frame_system_supertrait_check]`
 --> $DIR/step_007.rs:4:2
  |
4 |     pub trait Config {}
  |     ^^^
